mesonbuild/mesonlib.py

```diff
@@ -1127,3 +1057,27 @@ def __exit__(self, *args):
         elif have_msvcrt:
             msvcrt.locking(self.lockfile.fileno(), msvcrt.LK_UNLCK, 1)
         self.lockfile.close()
+
+class OrderedEnum(Enum):
+    """
+    An Enum which only offers homogeneous comparison. Taken from
+    https://docs.python.org/3/library/enum.html#orderedenum
```

Comment:
The page does not mention about licenses of code samples they have. This is supremely annoying because just copying stuff is not cool (even though they probably want people to do that).